src/EVM/SymExec.hs

```diff
@@ -34,6 +35,13 @@ import Data.Tuple (swap)
 import Data.Vector qualified as V
 import Data.Vector.Storable qualified as VS
 import Data.Vector.Storable.ByteString (vectorToByteString)
+import Control.Monad.State.Strict
+
+import Control.Concurrent.Chan (Chan, newChan, writeChan, readChan)
+import Control.Concurrent.STM (writeTChan, newTChan, TChan, readTChan, atomically, isEmptyTChan)
+import Control.Concurrent (forkIO, killThread)
+import Control.Concurrent.STM.TVar (TVar, newTVarIO, modifyTVar, readTVar, writeTVar, readTVarIO)
+import Control.Concurrent.STM.TMVar (putTMVar, takeTMVar, TMVar, putTMVar, takeTMVar, newEmptyTMVarIO)
```

Comment:
This should be moved before `Data`, to preserve alphabetical order.

Reply:
Yep, fixed!